python/pyspark/streaming/tests.py

```diff
@@ -1003,6 +1003,11 @@ def check_output(n):
         self.ssc.stop(True, True)
 
 
+# Must be same as the variable and condition defined in modules.py
+kafka_test_environ_var = "ENABLE_KAFKA_TESTS"
```

Comment:
I'd call this `ENABLE_KAFKA_0_8_TESTS`, ignoring my question about the existence of 0.10 tests in pyspark.